Feature/microvg documentation (#396)
MicroEJ/docs: ApplicationDeveloperGuide/UI/MicroVG/vectorfont.rst

- the text strings are scalable and can be transformed using a `Matrix` object.
- the TTF/OTF font files don't need to be preprocessed.
- the text strings can be drawn with opacity, a color or a linear gradient.

The library also considers the `Kerning` space described in the font file kerning table, and allows a fine adjustement of the inter-letters spacing.

Its also provides metrics measurement methods to correctly place the text within the surrounding drawing elements (i.e. in a label).

Loading a Font File
-------------------

Font files must be declared as ressources in a `.list` file available in the classpath(:ref:`chapter.microej.applicationResources`).

Then the font has to be loaded in a `VectorFont` object with a call to `ej.microvg.VectorFont.loadFont` <FIXME link to API>. This `VectorFont` object can then be used to draw text strings.

Text String Drawing
-------------------

A string can be drawn in the graphic context with a call to `ej.microvg.VectorGraphicsPainter.drawString` <FIXME link to API>.

The text string height is scalable, and multiple font files can be used in parrallel.

> |startTable| 

Author: I updated the table construction aliases to fix this issue.

Metrics and Text Positioning
----------------------------

All metrics provided by the `ej.microvg.VectorFont` class are given for a specific font size. The font size defines the height to which each character bounding box will be scaled.

The following figure presents some concepts of font metrics standarts:

.. figure:: images/fontMetrics.png
	:align: center

When a string is drawn with a call to `ej.microvg.VectorGraphicsPainter.drawString()`, the anchor point of the string is the top left corner of the text rendering box. This anchor point is located horizontally on the first pixel of the first drawn glyph and vertically on the max ascent line.

.. figure:: images/anchorPoint.png
	:align: center
	:width: 600px

The `ej.microvg.VectorFont.getBaselinePosition()` method can be used to position the text baseline on a horizontal line.

The `ej.microvg.VectorFont.getHeight()` method can be used to center a text inside a label, by positionning the anchor point in order to have the same space above and below the text string.

> Two other methods are available to position a known text in a label:

Reviewer: ```suggestion
Two other methods are interesting to position a given text within an area:
```
